Core/FileLoaders/LocalFileLoader.cpp

```diff
@@ -15,23 +15,25 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
-#include <cstdio>
 #include "file/file_util.h"
 #include "Common/FileUtil.h"
 #include "Core/FileLoaders/LocalFileLoader.h"
+#ifdef _WIN32
+#include "Common/CommonWindows.h"
+#endif
 
 LocalFileLoader::LocalFileLoader(const std::string &filename)
 	: fd_(0), f_(nullptr), filesize_(0), filename_(filename) {
+    // FIXME: perhaps at this point we should just use plain open?
```

Comment:
Well, would use CreateFile, yes, which is what `DirectoryFileSystem` uses too for the same reason.  Just need to use `ConvertUTF8ToWString()` on the argument.

Note there's some restriction on UWP, which I don't know much about, requiring CreateFile2:
https://github.com/hrydgard/ppsspp/blob/dd23588cd7e4efba35f8309182128a6a84194e60/Core/FileSystems/DirectoryFileSystem.cpp#L224

I assume OpenFile isn't UWP safe since I think it's generally deprecated.

-[Unknown]